test/tests/php-argon2/expected-std-out.txt

```diff
@@ -0,0 +1 @@
+bool(true)
```

Comment:
Instead of using `expected-std-out.txt`, I think it would make more sense to just use [`exit()`](https://secure.php.net/exit) here and let the non-zero exit code propagate for failure.

So for cases where Argon2 isn't expected to be supported, they can output a brief message and immediately `exit(0)`, which I think simplifies the code a little too (and makes it easier to debug IMO).

Reply:
That happens if you mindlessly convert the hello-world example, instead of verifying how the tests actually work :innocent: I'll take a look.